gradle.properties

```diff
@@ -14,4 +14,6 @@ android.useDeprecatedNdk=true
 BUTTERKNIFE_VERSION=8.6.0
 
 org.gradle.jvmargs=-Xmx1536M
+
+#TODO: Temporary disabled. https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#aapt2
```

Comment:
Sorry, could you please also link to the PR where we had this discussion so that others can refer back to it for context? Thanks!

Reply:
Done. :)